go/teams/loader.go

```diff
@@ -242,6 +250,10 @@ func (l *TeamLoader) load1(ctx context.Context, me keybase1.UserVersion, lArg ke
 		mungedForceRepoll = true
 		mungedWantMembers = nil
 	}
+	if !mungedForceRepoll && l.InForceRepollMode(ctx) {
+		l.G().Log.CDebugf(ctx, "Must force-repoll in force-repoll mode")
+		mungedForceRepoll = true
```

Comment:
is this used? Anyway throw this in `load2DecideRepoll` instead of munge

Reply:
cool thanks!

Reply:
done